### PR TITLE
Modified UWSGI config to ignore client disconnects errors.

### PR DESCRIPTION
**Pull request by adamjezek98**
Also, some sentry integrations were added.

### CODE REVIEW

File: src/karmen_backend/server/__init__.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __call__(self, *args, **kwargs):
 if os.environ.get("SENTRY_DSN") is not None:
     sentry_sdk.init(
         dsn=os.environ.get("SENTRY_DSN"),
-        integrations=[FlaskIntegration()],
+        integrations=[FlaskIntegration(), CeleryIntegration(), RedisIntegration()],
```
Comment (czervenka) [inline src/karmen_backend/server/__init__.py:43]: this was added by @pavel-mxsf  as well wasn't it?

Comment (czervenka) [reply src/karmen_backend/server/__init__.py:45]: I was right ... I resolved the conflict with Pavel's change.

**Comment by adamjezek98** [comment]
@czervenka please stop merging master into open PR's, it makes unpretty history, rebasing is much cleaner

**Comment by czervenka** [comment]
OK, sorry. I was hard trying to merge as much pull requests as possible. To push things a little bit forward.

**Pull request merged**